Makefile

```diff
@@ -8,13 +8,18 @@ TOOLS := \
 	oci-create-runtime-bundle \
 	oci-image-validate \
 	oci-unpack
+MAN := \
+	oci-create-runtime-bundle.1 \
+	oci-image-validate.1 \
+	oci-unpack.1
```

Comment:
> The MAN is consistent with TOOLS named.

Right, but using the substitution reference lets you define the pattern “all tools have a %.1 man page” instead of re-listing all the tool names.  That means there's only one place to touch (`TOOLS`) when you add/remove new commands.